test/interfaces.jl

```diff
@@ -0,0 +1,113 @@
+#using QEDprocesses
+using Random
+using Distributions
+using QEDbase
+
+RNG = MersenneTwister(708583836976)
+DIM = rand(RNG, 2:8)
```

Comment:
Maybe also cover the edge-case `DIM=1`.

Reply:
Added in c8a7e0dad3a11dcfd62337ab780529620313383f